packages/terra-paginator/package.json

```diff
@@ -50,6 +50,6 @@
     "props-table": "props-table ./src/Paginator.jsx --out-dir ./docs/props-table",
     "test": "npm run test:jest && npm run test:wdio",
     "test:jest": "jest --config ../../jestconfig.js",
-    "test:wdio": "../../node_modules/.bin/wdio ../../wdio.conf.js"
+    "test:wdio": "../../node_modules/.bin/wdio ../../node_modules/terra-dev-site/config/wdio/wdio.conf.js"
```

Comment:
Curious, why the change here?

Reply:
When I run npm install and try to run tests in this package, terra-paginator has trouble finding the wdio.conf.js file as it no longer exists. After looking through other packages, their wdio command looks like this, so a tool kit change must have impacted this.